packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -52,6 +52,11 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
           boxShadow: v.boxShadowPressed,
         },
       }),
+      ...(p.selected && {
+        backgroundColor: v.backgroundColorSelected,
+        borderColor: v.borderColorSelected,
+        boxShadow: v.boxShadowSelected,
```

Comment:
```suggestion
        backgroundColor: v.selectedBackgroundColor,
        borderColor: v.selectedBorderColor,
        boxShadow: v.selectedBoxShadow,
```

Should prop be first there?